### ISSUE TITLE
Crash in library with play-services-location v21.0.0

### ISSUE THREAD

**Issue by MRezaNasirloo** [Adyen/adyen-3ds2-android#49]
**Describe the bug**
We see a crash after upgrading play-services-location from 20.0.0 to 21.0.0
Adyen version: 2.2.9

here is the stack trace:

```
FATAL EXCEPTION: main
Process: com.my.app.debug, PID: 6768
java.lang.IncompatibleClassChangeError: Found interface com.google.android.gms.location.FusedLocationProviderClient, but class was expected (declaration of 'com.google.android.gms.location.FusedLocationProviderClient' appears in /data/app/~~lO1VCWgXkIerJpXxMj6fzA==/com.my.app.debug-8MKNbvc3XLju264KeOGN4Q==/base.apk!classes17.dex)
    at atd.m.g$c.a(Unknown Source:2)
    at atd.m.g.a(Unknown Source:7)
    at atd.m.g$e.d(Unknown Source:1)
    at atd.m.g$e.c(Unknown Source:1)
    at atd.i.d.a(Unknown Source:4)
    at atd.h.b.a(Unknown Source:28)
    at atd.h.b.<init>(Unknown Source:17)
    at atd.h.b.a(Unknown Source:3)
    at com.adyen.threeds2.internal.g.initialize(Unknown Source:14)
    at com.my.app.payment.processout.ProcessOutInteractor$getThreeDSHandler$1$doFingerprint$1.invokeSuspend(ProcessOutInteractor.kt:326)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
    at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
    at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
    at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
    at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
    at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
    Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@dbccb7d, Dispatchers.Main.immediate]
```

**To Reproduce**
1. Update play-services-location to v21
2. Initialized 3ds2 lib

**Expected behavior**
no crash

**Comment by tkuntubayev** [comment]
Hi, thanks for pointing this.
We will update the compatibility with play-services-location v 21.0.0 for the next release.

**Comment by tkuntubayev** [comment]
Hey @MRezaNasirloo,
We've just released a new version 2.2.10 with play-services-location v 21.0.0 compatibility.
Thank you again for indication!
